tools/markdown-checker/lib/analyzer/collectLinkDefinitions.js

```diff
@@ -2,14 +2,15 @@
  * @param {Object} AST
  * @returns {Object} - collected link definitions
  */
-module.exports = function(AST) {
-    let definitions = {};
-    AST.children.forEach(node => {
-      if (node.type === "definition") {
-        let label = node.label;
-        definitions[label] = node.url;
-      }
-    });
-    return definitions;
-  };
-  
\ No newline at end of file
+function collectLinkDefinitions(AST) {
+  const definitions = {};
+  AST.children.forEach((node) => {
+    if (node.type === 'definition') {
+      const { label } = node;
+      definitions[label] = node.url;
+    }
+  });
+  return definitions;
+}
+
+module.exports = collectLinkDefinitions;
```

Comment:
Parse süreci ve okunabilirlik için son satıra bi boşluk atıyor editör default olarak. Onu kaldırmayalım. Review da özellikle belirtilmişti.

Reply:
airbnb config'i de son satıra 1 boşluk atmayı zorunlu kılıyor. burası için ekledim ben tekrar.